automation/taskcluster/lib/tasks.py

```diff
@@ -39,12 +39,12 @@ def __init__(
         self.date = arrow.get(date_string)
         self.trust_level = trust_level
 
-    def craft_assemble_release_task(self, architectures, is_staging=False):
+    def craft_assemble_nightly_task(self, architectures, is_staging=False):
         artifacts = {
             'public/target.{}.apk'.format(arch): {
                 "type": 'file',
                 "path": '/opt/fenix/app/build/outputs/apk/'
-                        '{}Greenfield/release/app-{}-greenfield-release-unsigned.apk'.format(arch, arch),
+                        '{}/greenfieldNightly/app-{}-greenfieldNightly-unsigned.apk'.format(arch, arch),
```

Comment:
Nit: I just realized we can reuse https://github.com/mozilla-mobile/fenix/pull/1693/files#diff-59f90d509bc6ae2c9d877250c5ba49c5R458. We can expose a function called `_craft_apk_full_path_from_architecture_and_build_type()` and call it here. Can be done in a followup

Reply:
With the advent of `decisionlib`/`taskgraph`, I'll leave this for a follow-up